resources/style.css

```diff
@@ -0,0 +1,399 @@
+	position: absolute;
+}
+.wave-bottom {
+	position: absolute;
+}
+  .content {
+	 margin: auto;
+ }
+  .title {
+	 color: white;
+	 font-size: 3.5rem;
+ }
+ .text {
+	  color: white;
+	  font-size: 2.0rem; 
+	  font-weight: 100; 
+	  letter-spacing: 0.5px;
+  }
+  .line {
+  	 width: 80%;
+  }
+.button {
+	position: relative;
+	margin-top: 50px;
+}
+  .see-more {
+   	border: 0.2rem solid #fff;
+	text-decoration: none; 
+	color: rgb(247, 241, 241);
+   	padding: 12px 15px;
+   	cursor: pointer;
+	font-size: 1.2rem;
+	font-weight: bold;
+	border-radius: 8px;
+   }
+/*   BUTTON HOVER */
+    .see-more:hover {
+		  background:  rgb(255, 255, 255);
+		  color: rgb(194, 68, 68);
+		  font-weight: bold;
+	  }
+/* ABOUT ME  */
+.container-about-me {
+	width: 1200px;
+	margin: 200px auto 0 auto;
+	padding-top: 40px;
+}
+.content-about-me {
+	display: flex; 
+	justify-content: space-around;
+}
+.about-me-text h2 {
+	color: #202c2d;
+}
+.about-me-text {
+	right: 120px;
+	position: relative;
+	width: 40%;
+	font-size: 2rem;
+	text-align: justify;
+}
+.about-me-text p {
+	letter-spacing: .5px;
+	overflow-wrap: break-word;
+	word-wrap: break-word;
+	hyphens: auto;
+}
+.headshot {
+	width: 230px;
+    height: 300px;
+}
+ .headshot img {
+	box-shadow: 13px 9px 15px -1px rgba(63, 62, 62, 0.32);
+	border:1px solid #000000;
+	border-radius: 8px;
+	width: 100%;
+	height: 100%;
+	position: relative;
+	margin-top: 80px;
+}
+/*	 PORTFOLIO  */ 
+	  .portfolio-heading {
+		  width:  1200px;
+		  padding-top: 30px;
+		  margin: 110px auto 0 auto;
+		  color: #202c2d;
+		  font-size: 2rem;
+	  }
+	  .container-portfolio {
+		border: rgb(185, 177, 177) 1px solid;
+		box-shadow: 13px 9px 15px -1px rgba(63, 62, 62, 0.32);
+		border-radius: 8px;
+		width: 1200px;
+	    margin: 40px auto;
+	  }
+	  .portfolio-flex {
+		  margin: 40px 40px 15px 40px;
+		  display: flex;
+		  justify-content: space-around; 
+	  }
+	   .col-img {
+		 width: 380px;
+		 height: 280px;
+	  }
+	 .col-img img {
+		  width: 100%;
+		  height: 100%;
+		  border-radius: 8px;
+	  }
+	  .col-text {
+		  font-size: 1.8rem;
+		  max-width: 40%;
+	  }	
+	  .portfolio-links {
+		  display:flex;
+		  justify-content: flex-end;
+		  position: relative;
+		  bottom: 30px;
+		  right:30px;
+	  }
+	  .left-link {
+		left:50px;
+		justify-content: flex-start;
+	  }
+	  .git { 
+		text-decoration: none;
+		margin-right:30px;
+		font-size: 2rem;
+		color: #000;
+	  }
+	  .git span {
+		  font-size: 1.5rem;
+		  padding-right: 8px;
+	  }
+	  .git:hover {
+		  color: rgb(194, 68, 68);;
+	  }
+	/*  ARROWS    */
+	.arrow-holder {
+		position: relative;
+		margin-top: 120px;
+		margin-left: 120px;
+		width: 220px;
+	  }
+	  .arrow {
+		float: left;
+		margin-right: 15px;
+		width: 0; 
+		border-top: 10px solid transparent;
+		border-bottom: 10px solid transparent;
+	  }
+	  .arrow-right {
+		  border-left: 15px solid black;
+	  }
+	  .arrow-left {
+		  border-right: 15px solid black;
+	  }
+	  @-webkit-keyframes flash {
```

Comment:
why only webkit keyframes? this wont animate on non-webkit browsers. just use `@keyframes`